seednsort.md

```diff
@@ -23,4 +23,4 @@ The Seeders/Sorters team will use the EDGI subprimer systems ([found here](https
   - In the spreadsheet is automatically associated with a universal unique identifyer (UUID) that was generated in advance.
 
```

Comment:
I thought I had merged it in somewhere... sorry, a couple bullet points above
- You can check whether the page or some files are rendered using the Internet Archive's [Wayback Machine Chrome Extension](https://chrome.google.com/webstore/detail/wayback-machine/fpnmgdkabkmnadcjpehmlllkndpkmiak)